Makefile

```diff
@@ -27,12 +28,14 @@ validate:
 
 1.10: validate
 	packer build \
+		-var aws_region=$(AWS_DEFAULT_REGION) \
```

Comment:
Variable name needs to be updated here and on line 38 as well. (`AWS_REGION`)

Reply:
Sorry I've missed this two.